charts/tekton-pipelines/values.yaml

```diff
@@ -111,6 +111,9 @@ buildpacks:
             fi
           securityContext:
             privileged: true
+      # -- steps to run in the `pre-deploy` task prior to ArgoCD sync command
+      # can be useful to prepare different backups and tests before real deploy
+      preDeployTaskSteps: []
```

Comment:
add sample values for this arg below this line, but commented - like I do in my tasks on addon values. So anyone can get a quick reference as our README.md for the chart is quite big

Reply:
for all below as well

Reply:
done